pgs_point.sql.in

```diff
@@ -167,3 +167,8 @@ CREATE OPERATOR  <-> (
 COMMENT ON OPERATOR <-> (spoint, spoint) IS
   'distance between spherical points';
 
+CREATE FUNCTION get_gravity_center(dots_vector SPoint[])
```

Comment:
My initial reaction is that the function should be called `gravity_center` instead of `get_gravity_center`. I could be wrong about that though. Is there a convention or precedent for how functions should be named?

Reply:
I propose to name it 'centroid'